src/UI/Utils/Element.elm

```diff
@@ -179,6 +179,8 @@ ellipsis : List (Attribute msg)
 ellipsis =
```

Comment:
I'm not sure if it makes sense to have this function "public" and also outside of Internal.Text 🤔